tools/TerminalVelocity/test/TerminalVelocityFeatures-Test-Experimental.cs

```diff
@@ -0,0 +1,273 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+// THIS FILE IS AUTOMATICALLY GENERATED; DO NOT EDIT IT
+
+// INPUT FILE: TerminalVelocityFeatures-Test.xml
+
+// Feature constants
+//DISABLED: WINDOWSAPPSDK_FEATURE_ALWAYSDISABLED_ENABLED
+//DISABLED: WINDOWSAPPSDK_FEATURE_ALWAYSDISABLED_ALWAYSENABLEDCHANNELS_EXPERIMENTAL_ENABLED
+//DISABLED: WINDOWSAPPSDK_FEATURE_ALWAYSDISABLED_ALWAYSENABLEDCHANNELS_EXPERIMENTALPREVIEW_ENABLED
+//DISABLED: WINDOWSAPPSDK_FEATURE_ALWAYSDISABLED_ALWAYSENABLEDCHANNELS_EXPERIMENTALPREVIEWSTABLE_ENABLED
+//DISABLED: WINDOWSAPPSDK_FEATURE_ALWAYSDISABLED_ALWAYSENABLEDCHANNELS_EXPERIMENTALPREVIEWWINDOWSINBOX_ENABLED
+//DISABLED: WINDOWSAPPSDK_FEATURE_ALWAYSDISABLED_ALWAYSENABLEDCHANNELS_EXPERIMENTALSTABLE_ENABLED
+//DISABLED: WINDOWSAPPSDK_FEATURE_ALWAYSDISABLED_ALWAYSENABLEDCHANNELS_EXPERIMENTALSTABLEWINDOWSINBOX_ENABLED
+//DISABLED: WINDOWSAPPSDK_FEATURE_ALWAYSDISABLED_ALWAYSENABLEDCHANNELS_EXPERIMENTALWINDOWSINBOX_ENABLED
+//DISABLED: WINDOWSAPPSDK_FEATURE_ALWAYSDISABLED_ALWAYSENABLEDCHANNELS_PREVIEW_ENABLED
+//DISABLED: WINDOWSAPPSDK_FEATURE_ALWAYSDISABLED_ALWAYSENABLEDCHANNELS_PREVIEWSTABLEWINDOWSINBOX_ENABLED
+//DISABLED: WINDOWSAPPSDK_FEATURE_ALWAYSDISABLED_ALWAYSENABLEDCHANNELS_STABLE_ENABLED
+//DISABLED: WINDOWSAPPSDK_FEATURE_ALWAYSDISABLED_ALWAYSENABLEDCHANNELS_WINDOWSINBOX_ENABLED
+//DISABLED: WINDOWSAPPSDK_FEATURE_ALWAYSDISABLED_ID_ENABLED
+#Define WINDOWSAPPSDK_FEATURE_ALWAYSENABLED_ENABLED
+//DISABLED: WINDOWSAPPSDK_FEATURE_ALWAYSENABLED_ALWAYSDISABLEDCHANNELS_EXPERIMENTAL_ENABLED
+//DISABLED: WINDOWSAPPSDK_FEATURE_ALWAYSENABLED_ALWAYSDISABLEDCHANNELS_EXPERIMENTALPREVIEW_ENABLED
+//DISABLED: WINDOWSAPPSDK_FEATURE_ALWAYSENABLED_ALWAYSDISABLEDCHANNELS_EXPERIMENTALPREVIEWSTABLE_ENABLED
+//DISABLED: WINDOWSAPPSDK_FEATURE_ALWAYSENABLED_ALWAYSDISABLEDCHANNELS_EXPERIMENTALPREVIEWWINDOWSINBOX_ENABLED
+//DISABLED: WINDOWSAPPSDK_FEATURE_ALWAYSENABLED_ALWAYSDISABLEDCHANNELS_EXPERIMENTALSTABLE_ENABLED
+//DISABLED: WINDOWSAPPSDK_FEATURE_ALWAYSENABLED_ALWAYSDISABLEDCHANNELS_EXPERIMENTALSTABLEWINDOWSINBOX_ENABLED
+//DISABLED: WINDOWSAPPSDK_FEATURE_ALWAYSENABLED_ALWAYSDISABLEDCHANNELS_EXPERIMENTALWINDOWSINBOX_ENABLED
+#Define WINDOWSAPPSDK_FEATURE_ALWAYSENABLED_ALWAYSDISABLEDCHANNELS_PREVIEW_ENABLED
```

Comment:
Because you're looking at the unit test. The feature names convey what they're exercising, they're very not-normal names anyone would ever use. Here's a better example given `dev\common\TerminalVelocityFeatures-DynamicDependency.xml`

Command line: `tools\TerminalVelocity\Generate-TerminalVelocityFeatures.cmd dev\Common\TerminalVelocityFeatures-DynamicDependency.xml -Namespace Microsoft.Windows.ApplicationModel.DynamicDependency -Language C++`

Output
```
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License. See LICENSE in the project root for license information.

// THIS FILE IS AUTOMATICALLY GENERATED; DO NOT EDIT IT

// INPUT FILE: dev\Common\TerminalVelocityFeatures-DynamicDependency.xml
// OPTIONS: -Channel Experimental -Namespace Microsoft.Windows.ApplicationModel.DynamicDependency

// Feature constants
#define WINDOWSAPPSDK_MICROSOFT_WINDOWS_APPLICATIONMODEL_DYNAMICDEPENDENCY_FEATURE_GENERATIONID_ENABLED 1

#if defined(__cplusplus)

namespace Microsoft::Windows::ApplicationModel::DynamicDependency
{

__pragma(detect_mismatch("ODR_violation_WINDOWSAPPSDK_MICROSOFT_WINDOWS_APPLICATIONMODEL_DYNAMICDEPENDENCY_FEATURE_GENERATIONID_ENABLED_mismatch", "AlwaysEnabled"))
struct Feature_GenerationId
{
    static constexpr bool IsEnabled() { return WINDOWSAPPSDK_MICROSOFT_WINDOWS_APPLICATIONMODEL_DYNAMICDEPENDENCY_FEATURE_GENERATIONID_ENABLED == 1; }
};

} // namespace Microsoft.Windows.ApplicationModel.DynamicDependency

#endif // defined(__cplusplus)
```

Command Line: `tools\TerminalVelocity\Generate-TerminalVelocityFeatures.cmd dev\Common\TerminalVelocityFeatures-DynamicDependency.xml -Namespace Microsoft.Windows.ApplicationModel.DynamicDependency -Language C#`

Output
```
Microsoft.Windows.ApplicationModel.DynamicDependency -Language C#
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License. See LICENSE in the project root for license information.

// THIS FILE IS AUTOMATICALLY GENERATED; DO NOT EDIT IT

// INPUT FILE: dev\Common\TerminalVelocityFeatures-DynamicDependency.xml
// OPTIONS: -Channel Experimental -Namespace Microsoft.Windows.ApplicationModel.DynamicDependency

// Feature constants
#Define WINDOWSAPPSDK_MICROSOFT_WINDOWS_APPLICATIONMODEL_DYNAMICDEPENDENCY_FEATURE_GENERATIONID_ENABLED

namespace Microsoft.Windows.ApplicationModel.DynamicDependency
{

public static class Feature_GenerationId
{
#if WINDOWSAPPSDK_MICROSOFT_WINDOWS_APPLICATIONMODEL_DYNAMICDEPENDENCY_FEATURE_GENERATIONID_ENABLED
    public const bool IsEnabled = true;
#else
    public const bool IsEnabled = false;
#endif
};

} // namespace Microsoft.Windows.ApplicationModel.DynamicDependency
```